### PR TITLE
JSONException issue in description variable

### PR DESCRIPTION
**Pull request by stachu540**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

## Description

Discord last time assign [those variables](https://discordapp.com/developers/docs/topics/oauth2#get-current-application-information) to nullabe. Since [this one](https://pastebin.com/57Dw1wh5). Those minimal changes only change nullable `description` to be a `null`

### CODE REVIEW

**Comment by Sanduhr32** [comment]
I would like to know if it's consistent `null` or an empty string from Discord. In any case it should be a consistent null or empty string in jda.

File: src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java [hunk 1]
```diff
@@ -1286,7 +1286,7 @@ else if (channelType.isGuild())
 
     public ApplicationInfo createApplicationInfo(JSONObject object)
     {
-        final String description = object.getString("description");
+        final String description = object.optString("description", null);
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java]: This should be `optString("description, "")`

**Comment by kantenkugel** [comment]
If this is in any way similar to the channel description changes they did a while back, then yes, it can be either null or an empty string... and i feel like the empty string is the more feasible default in those cases then

File: src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java (hunk 1 above)
Comment (stachu540) [reply src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java]: Wish granted

**Comment by stachu540** [comment]
Confirm updated referenced PR discordapp/discord-api-docs#814
image
Closing this PR

**Pull request closed without merging**